aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -319,6 +319,13 @@ Tensor& remainder_(Tensor& self, const Tensor& other) {
 }
 
 Tensor& floor_divide_out(Tensor& result, const Tensor& self, const Tensor& other) {
+  TORCH_WARN_ONCE(
+    "floor_divide rounding will change in a future version of pytorch. "
```

Comment:
"floor_divide is deprecated, and it will be removed in a future version of PyTorch. It currently..."